CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributor Manual
+
+We welcome contributions of any size and skill level. As an open source project, we believe in giving back to our contributors and are happy to help with guidance on PRs, technical writing, and turning any feature idea into a reality.
+
+> **Tip for new contributors:**
+> Take a look at [https://github.com/firstcontributions/first-contributions](https://github.com/firstcontributions/first-contributions) for helpful information on contributing
+
+This document is a work in progress! Most information on contributing can be found in the main README.md file. But, we need a place to begin to document some of our tools and processes, like our `Aside` Markdown feature.
+
+## Writing Asides (aka how not to abuse `blockquote`)
+
+This is an accessible component, based on the [recommended markup from the BBC’s GEL design system](https://bbc.github.io/gel/components/breakout-boxes/#recommended-markup).
+
+The component has **note**, **tip**, **caution** and **danger** variants with colour, iconography, and default labelling distinct for each.
+
+This is an Astro integration/remark plugin that allows us to use a simple syntax to use the component in Markdown and also avoid needing to import it in the frontmatter all the time.
```

Comment:
Here’s a little more context for aside to try to guide people on when to reach for it.

```suggestion
Sometimes in documentation you want to provide information that is complementary but not strictly part of the current text or call out something that is particularly important. For example, maybe you want to include a tip that isn’t essential but could be helpful or warn a reader about a potential pitfall.

For these use cases you can use our aside component. This is an accessible component, based on the [recommended markup from the BBC’s GEL design system](https://bbc.github.io/gel/components/breakout-boxes/#recommended-markup).

The component has **note**, **tip**, **caution** and **danger** variants with colour, iconography, and default labelling distinct for each.

You can use a simple custom syntax to use the component in Markdown and also avoid needing to import it in the frontmatter all the time.
```